.github/workflows/ci.yml

```diff
@@ -171,8 +171,6 @@ jobs:
 
       - run: uv sync --only-dev
```

Comment:
Should we also move the `${{ matrix.install.command }}` here and make the test command `uv run --no-sync ...`?